per/serializers.py

```diff
@@ -1011,6 +1021,18 @@ def validate(self, data):
             )
         return data
 
+    def validate_per(self, per):
+        if per is None:
+            raise serializers.ValidationError("This field is required")
+        country_per = list(
+            Country.objects.filter(id=self.initial_data['country']).values_list("per_overviews", flat=True)
+        )
+        if per and per.id not in country_per:
+            raise serializers.ValidationError(
+                gettext(f"Per {per.id} doesn't match country {self.initial_data['country']}")
```

Comment:
F-string doesn't work with Django translation

https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#internationalization-in-python-code
```
    output = _('Today is %(month)s %(day)s.') % {'month': m, 'day': d}
```